### DISCUSSION TITLE
Created date is not saving into filename correctly? bug?

### DISCUSSION THREAD

**Discussion by BlankSlate56** [jonaswinkler/paperless-ng#599] [Support]
I just noticed that my Paperless_filename_format variable is not saving my filenames correctly.  Below is my variable
![image](https://user-images.githubusercontent.com/78878227/108732253-008c4600-74f3-11eb-9a77-8198a9d2721d.png)

As stated in other threads, I have been manually updating the Created date when necessary.  I just noticed in my saved files in the paperless output that the date in the file resulting from the filename variable is actually 1 day later than what the created date says.  In the pics below, I have entered and saved the Created date as 12/01/2020, but it is saving the date in the filename as 12/02/2020.  Why is this?

Created date in paperless: 12/01/2020
![image](https://user-images.githubusercontent.com/78878227/108732620-58c34800-74f3-11eb-80b1-936f4a953a95.png)

Saved Filename: 12/02/2020
![image](https://user-images.githubusercontent.com/78878227/108732745-785a7080-74f3-11eb-865d-2d82639e1dab.png)

**Comment by jonaswinkler** [comment]
Probably something related to timezones as well... This is a continuation project and honestly, I have no idea why they decided to store both date and time for this field in the database; and also enable timezone support for that matter. This is causing all that trouble now. Storing just the date in the database should have been enough, but now it's pretty hard to change that.

I'll look into this. What's your time zone?

**Reply by BlankSlate56** [reply]
I'm in US-Chicago/Central time.  I understand that it is a continuation and hope you don't take my posts as complaints.  I look through previous discussions prior to posting and haven't found similar issues, so I didn't know if it was just an issue with my setup or what was going on.  Thanks for looking into it.
